README.md

```diff
@@ -47,12 +47,17 @@ buildscript {
         classpath 'com.novoda:gradle-static-analysis-plugin:0.5.1'
     }
 }
-```
 
-You can then apply the plugin via:
+apply plugin: 'com.novoda.static-analysis'
+```
+        
+or from the [Gradle Plugins Repository](https://plugins.gradle.org/):
 
 ```gradle
-apply plugin: 'com.novoda.static-analysis'
+plugins {
+    id 'com.novoda:gradle-static-analysis-plugin' version '0.5.1'
```

Comment:
:x: What about keeping both consistent? 
I think this should be `com.novoda.static-analysis`.
Also `gradle` and `plugin` words are redundant in this context. Having them in the repo name makes sense tho but maybe not in this name here.

Reply:
@tasomaniac has a good point here: I believe this needs to follow the naming we use with `apply` method, so in this case `com.novoda.static-analysis`.

Reply:
yep, I agree 💯